c2rust-analyze/src/analyze.rs

```diff
@@ -993,7 +993,10 @@ fn run(tcx: TyCtxt) {
                 if !node_info.unique {
                     perms.remove(PermissionSet::UNIQUE);
                 }
-                // TODO: PermissionSet::NON_NULL
+                if let Some(true) = g.is_null {
```

Comment:
`is_null` should always be `Some` here, right?  Maybe this should use `unwrap()`.